pybliometrics/scopus/utils/startup.py

```diff
@@ -1,21 +1,81 @@
-import configparser
+from configparser import ConfigParser
 from collections import deque
 from pathlib import Path
+from typing import List, Dict, Optional, Type
+import os
+from warnings import warn
```

Comment:
Always sort alphabetically: the full imports on top, the partial imports on the bottom.